src/System.Text.Json/tests/Serialization/EnumConverterTests.cs

```diff
@@ -0,0 +1,117 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+using Xunit;
+
+namespace System.Text.Json.Serialization.Tests
```

Comment:
Please add a test that uses `[JsonConverter]` to specify the stringenum converter for a single property, but keeping the default converter as-is.

Reply:
Ok

Reply:
Apparently the converter attribute does not work with factories.